.github/workflows/cpp.yaml

```diff
@@ -7,11 +7,72 @@ on:
   pull_request:
 
 jobs:
+  do_cpp:
+    if: |
+      github.event_name != 'pull_request' ||
+      (
+        !contains(github.event.pull_request.body, '[no C++]')
```

Comment:
I would prefer all lower case, but fine by me if we have to following lower+upper.  Is this case sensitive; can you make contains to be case insensitive

Reply:
It's case insensitive. :-) I'll make sure that fact is in the docs.